_data/conferences.yml

```diff
@@ -2654,3 +2654,16 @@
   cfp_phrase: CFP closes
   cfp_date: 2024-06-01
   cfp_link: https://reddotrubyconf.com/papers/new
+
+- name: Kaigi on Rails 2024
+  location: Tokyo, Japan
+  start_date: 2024-10-25
+  end_date: 2024-10-26
+  url: https://kaigionrails.org
+  twitter: kaigionrails
+  mastodon: https://ruby.social/@kaigionrails
+
+- name: Ruby Retreat 2024
```

Comment:
Thank you @hsbt! Is there any more information about the Ruby Retreat? I somehow couldn't find anything online

Reply:
Ah, I see. Thank you for the clarification! 🙏🏼

Reply:
Though, I just realized, we could add the Retreat with a  `date_precision: month` so that it at least shows up as "October 2024":

```yml
name: Ruby Retreat 2024
location: Warrnambool, Victoria, Australia
date_precision: month
start_date: 2024-10-31
end_date: 2024-10-31
url: https://docs.google.com/forms/d/e/1FAIpQLSf-vA6L9UJV3IpBkfl5HqBb6pat1BIz62Q6JIKeLMD8d_dTlQ/viewform
```